api_app/models/domain/resource.py

```diff
@@ -41,16 +23,17 @@ class Resource(AzureTREModel):
     templateName: str = Field(title="Resource template name", description="The resource template (bundle) to deploy")
     templateVersion: str = Field(title="Resource template version", description="The version of the resource template (bundle) to deploy")
     properties: dict = Field({}, title="Resource template parameters", description="Parameters for the deployment")
-    deployment: Deployment = Field(Deployment(status=Status.NotDeployed, message=""), title="Deployment", description="Fields related to deployment of this resource")
+    isActive: bool = Field(True, title="Is Active", description="Is the resource active? Will be False when deleted.")
```

Comment:
Have we still got the concept of disabled?

A resource had to be disabled before being deleted.

Reply:
Disabled is in the json `properties` - here: https://github.com/microsoft/AzureTRE/blob/0300d32ef29616dc89a34c0362cf35c91f2a95b2/api_app/models/domain/resource.py#L47. The `IsActive` flag doesn't affect this logic.

Reply:
Thanks.